rpc/src/rpc_health.rs

```diff
@@ -54,84 +52,74 @@ impl RpcHealth {
             }
         }
 
+        if self.override_health_check.load(Ordering::Relaxed) {
+            return RpcHealthStatus::Ok;
+        }
         if !self.startup_verification_complete.load(Ordering::Acquire) {
             return RpcHealthStatus::Unknown;
         }
 
-        if self.override_health_check.load(Ordering::Relaxed) {
```

Comment:
noticed we lost support for this flag.  assuming it was added instead of fixing the root cause, but should probably restore and properly deprecate if that's the case

Reply:
ah :facepalm: i only checked _down_ with the rest of the new logic